src/main/cms/cms_menu_persistent_stats.c

```diff
@@ -0,0 +1,107 @@
+/*
+ * This file is part of Cleanflight and Betaflight.
+ *
+ * Cleanflight and Betaflight are free software. You can redistribute
+ * this software and/or modify this software under the terms of the
+ * GNU General Public License as published by the Free Software
+ * Foundation, either version 3 of the License, or (at your option)
+ * any later version.
+ *
+ * Cleanflight and Betaflight are distributed in the hope that they
+ * will be useful, but WITHOUT ANY WARRANTY; without even the implied
+ * warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this software.
+ *
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include <stdbool.h>
+#include <stdint.h>
+#include <string.h>
+#include <ctype.h>
+
+#include "platform.h"
+
+#ifdef USE_CMS
+#ifdef USE_PERSISTENT_STATS
+
+#include "cms/cms.h"
+#include "cms/cms_types.h"
+#include "cms/cms_menu_persistent_stats.h"
+
+#include "config/config.h"
+#include "pg/stats.h"
+
+uint32_t stats_total_flights;
+uint32_t stats_total_time_s;
+uint32_t stats_total_dist_m;
+int8_t stats_min_armed_time_s;
+
+static const void *cmsx_PersistentStats_onEnter(displayPort_t *pDisp)
+{
+    UNUSED(pDisp);
+
+    stats_total_flights = statsConfig()->stats_total_flights;
+    stats_total_time_s = statsConfig()->stats_total_time_s;
+    stats_total_dist_m = statsConfig()->stats_total_dist_m;
+    stats_min_armed_time_s = statsConfig()->stats_min_armed_time_s;
```

Comment:
I think the display looks a bit confusing as it combines stats and settings in one list - maybe add a `--- SETTINGS ---` or so above this line?

Reply:
@mikeller, I agree